modules/data-entry/src/main/frontend/src/questionnaire/BooleanQuestion.jsx

```diff
@@ -56,7 +56,8 @@ import AnswerComponentManager from "./AnswerComponentManager";
 //   unknownLabel="Does not compute"
 //   />
 function BooleanQuestion(props) {
-  const {classes, ...rest} = props;
+  const {existingAnswer, isEdit, classes, ...rest} = props;
```

Comment:
`existingAnswer` and `isEdit` don't need to be named here.
```suggestion
  const {classes, ...rest} = props;
```